gammapy/data/event_list.py

```diff
@@ -335,7 +336,7 @@ def select_energy(self, energy_range):
         Parameters
         ----------
         energy_range : `~astropy.units.Quantity`
-            Energy range ``[energy_min, energy_max)``
+            Energy range ``[energy_min, energy_max)``.
```

Comment:
Yes I notice that. It happens several times, so I figured that this must be to signify that energy_max is excluded from the band. In math notation it would be `[energy_min, energy_max[`. But maybe it is just a mistake. 
However if it is not a mistake, what should I do about it ?